docs/cdp/cdp_guide.md

```diff
@@ -45,7 +45,7 @@ This approach helps you create a single customer view.
 
 To create a personalized customer experience, you need to group your clients into specified audiences.
 [[= product_name =]] comes with a ready solution - segment groups.
-Segment group information is reused by various [[= product_name =]] functionalities, such as [Personalization](https://doc.ibexa.co/en/latest/personalization/personalization_brochure/) or content targeting.
+Segment group information is reused by various [[= product_name =]] functionalities, such as [Personalization](https://doc.ibexa.co/en/latest/personalization/personalization_product_guide/) or content targeting.
```

Comment:
Is there a reason for not using magic link here?

```suggestion
Segment group information is reused by various [[= product_name =]] functionalities, such as [Personalization](personalization_product_guide.md) or content targeting.
```

Reply:
No, just a mistake 😄